datatypes/src/main/java/org/hyperledger/besu/datatypes/TransactionType.java

```diff
@@ -28,6 +28,8 @@ public enum TransactionType {
   EIP1559(0x02),
   /** Blob transaction type. */
   BLOB(0x03),
+  /** Optimism Deposit transaction type. */
+  OPTIMISM_DEPOSIT(0x7e),
```

Comment:
idk if it will be easy but I will prefer to have another transaction type list for optimism in order to see more easilty the type of transaction by network

Reply:
did you had time to look at this point ?